src/lib/y2caasp/dhcp_ntp_servers.rb

```diff
@@ -41,5 +40,35 @@ def dhcp_ntp_servers
 
       Yast::LanItems.dhcp_ntp_servers.values.flatten.uniq
     end
+
+    #
+    # Propose the NTP servers from the DHCP response, fallback to a random
+    # machine from the ntp.org pool if enabled in control.xml.
+    #
+    # @return [Array<String>] proposed NTP servers, empty if nothing suitable found
+    #
+    def ntp_servers
+      # TODO: use Yast::NtpClient.ntp_conf if configured
+      # to better handle going back
+      servers = dhcp_ntp_servers
```

Comment:
dhcp_ntp_servers is still ok? It relies on LanItems and probably need to be updated to obtain the information from the new y2network classes.

Reply:
We decided to use as it is and report a separate bug in case there is something already broken or that needs to be fixed.